src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -179,6 +183,13 @@ private void ProcessProjectEvent(string messageJson)
                 });
             _plantSetter.SetPlant(projectEvent.Plant);
             _invitationRepository.UpdateProjectOnInvitations(projectEvent.ProjectName, projectEvent.Description);

```

Comment:
I noticed that _invitationRepository.UpdateProjectOnInvitations actually do nothing ... do we understand why? Should we throw the method?